README.md

```diff
@@ -18,47 +16,62 @@
 
-💻 **Enterprise-ready solutions**
+### 💻 Enterprise-ready solutions
 
-- Implements role-based access control (RBAC) for scalable role authorization, catering to a wide range of use cases.
-- Facilitates user management and provides audit logs for understanding identity-related user information and maintaining security.
-- Enables single sign-on (SSO) and multi-factor authentication (MFA) without extra coding.
-- Leverages Logto organizations to build multi-tenancy apps with ease.
+- Implements [Role-Based Access Control (RBAC)](https://docs.logto.io/docs/recipes/rbac/) for scalable authorization.
+- Enables user management with [audit logs](https://docs.logto.io/docs/recipes/inspect-audit-logs/) to track identity-related activities and maintain security.
+- Enables [single sign-on (SSO)](https://docs.logto.io/docs/recipes/single-sign-on/) and [multi-factor authentication (MFA)](https://docs.logto.io/docs/recipes/multi-factor-auth/) without minimal coding.
+- Leverage [Logto organizations](https://docs.logto.io/docs/recipes/organizations/understand-how-it-works/) to build [multi-tenancy](https://blog.logto.io/tenancy-models/) apps with ease.
 
-In a more approachable way, we refer to this solution as "[Customer Identity Access Management (CIAM)](https://en.wikipedia.org/wiki/Customer_identity_access_management)" or simply, the "Customer Identity Solution."
+## Customer Identity Access Management (CIAM) introductory courses
+For a more approachable introduction, check out Logto’s ***Customer Identity Access Management (CIAM) introductory courses***:
+- [CIAM 101](https://docs.logto.io/ciam-101-intro-authn-sso/): Authentication, Identity, SSO
+- [CIAM 102](https://blog.logto.io/ciam-102-authz-and-rbac/): Authorization & Role-based Access Control
 
-> [!IMPORTANT]
-> [Subscribe to us](https://logto.io/subscribe/?utm_source=github&utm_medium=repo_logto) now to stay updated with the latest information about the Logto Cloud (SaaS) and receive feature updates in real-time.
-
-## Get started
-
-- Visit our 🎨 [website](https://logto.io/?utm_source=github&utm_medium=repo_logto) for a brief introduction if you are new to Logto.
-- A step-by-step guide is available on 📖 [docs.logto.io](https://docs.logto.io/?utm_source=github&utm_medium=repo_logto).
+## Deep dives 
+For a ***deeper understanding*** of key identity management topics, explore our [blog](https://blog.logto.io/):
+- [Understanding Token Exchange in OAuth/OIDC](https://blog.logto.io/token-exchange)– Learn about token-based authentication systems.
+- [Opaque Token vs JWT](https://blog.logto.io/opaque-token-vs-jwt) – Discover the differences between token types for authentication.
+- [When Should I Use JWTs?](https://blog.logto.io/when-should-i-use-jwts) – A guide on when to use JWT for secure authentication.
+- [Bring Your Own Sign-In UI](https://blog.logto.io/bring-your-own-ui)– Customize your UI with Logto Cloud for flexible authentication.
```

Comment:
Should we also use sentense case for these external links? E.g., `Bring your own sign-in UI`

Reply:
Changed external links to match sentence case